server/models/Experiment.ts

```diff
@@ -0,0 +1,37 @@
+import { DataTypes, Model } from "sequelize"
+import sequelize from "../utils/sequelize"
+
+const experimentStatusEnum = DataTypes.ENUM("Draft", "Submitted", "Accepted")
+class Experiment extends Model {}
+
+Experiment.init(
+  {
+    // Model attributes are defined here
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
```

Comment:
Sorry for finding something else again, but I think it would be a good idea to also have a uniqe slug (basically the title just converted to a string which can be part of an url) so for example an experiment with the name "Optik Experiment Schüler" would be mapped to the slug "optik-experiment-schuler" and the slug would be one a way to identify the experiment and look better in the url of the browser than an id. (But we still need the id because the slug could change and references need a constant id). 
So could you just add s slug which is a string and unique and not null.

Reply:
added slug